sphinx_simplepdf/builders/simplepdf.py

```diff
@@ -172,6 +173,45 @@ def _toctree_fix(self, html):
             for link in links:
                 link["href"] = link["href"].replace(f"{self.app.config.root_doc}.html", "")
 
+            print(links)
```

Comment:
are the print() statements needed?

Reply:
I left one to tell the user if a duplicate headline has been found. others have been removed